sklearn/ensemble/_bagging.py

```diff
@@ -227,11 +227,11 @@ def fit(self, X, y, sample_weight=None):
             The training input samples. Sparse matrices are accepted only if
             they are supported by the base estimator.
 
-        y : array-like of shape (n_samples,)
+        y : ndarray of shape (n_samples,)
             The target values (class labels in classification, real numbers in
             regression).
 
-        sample_weight : array-like of shape (n_samples,), default=None
+        sample_weight : ndarray of shape (n_samples,), default=None
```

Comment:
same

Reply:
Should not `sample_weight` be `ndarray`?

Reply:
No it can also be a list, in which case it will be cast to an array internally.

Reply:
Can we leave `sample_weight`, to maintain uniformity across the `classes`?

Reply:
If you can revert it to how it was before it would be better.

 We are also trying to reduce code churn as it makes using git blame harder and I can't say that this particular line is a net positive, even if it doesn't matter too much either way.

Reply:
ok, I will do it.